tsl/test/sql/compress_bloom_sparse.sql

```diff
@@ -400,4 +400,136 @@ drop table corner;
 drop table badtable;
 drop table byref;
 
+-- Test configurable sparse indexes settings
+CREATE VIEW settings AS SELECT * FROM _timescaledb_catalog.compression_settings ORDER BY upper(relid::text) COLLATE "C";
+
+create table test_settings(x int, value text, u uuid, ts timestamp);
+select create_hypertable('test_settings', 'x');
+alter table test_settings set (timescaledb.compress,
+    timescaledb.compress_orderby = 'x',
+    timescaledb.compress_bloom = 'u',
+    timescaledb.compress_minmax = 'ts');
+
+select * from settings;
+
+-- same column
+alter table test_settings set (timescaledb.compress,
+    timescaledb.compress_orderby = 'x',
+    timescaledb.bloom = 'u',
+    timescaledb.minmax = 'u');
+select * from settings;
+
+--multi column
+alter table test_settings set (timescaledb.compress,
+    timescaledb.compress_orderby = 'x',
+    timescaledb.bloom = 'u,ts',
+    timescaledb.minmax = 'u,ts');
+select * from settings;
+
+--test errors
+\set ON_ERROR_STOP 0
+-- invalid syntax
+alter table test_settings set (timescaledb.compress,
+    timescaledb.compress_orderby = 'x',
+    timescaledb.compress_bloom = 'u, count(u)',
+    timescaledb.compress_minmax = 'ts');
+
+-- duplicate column
+alter table test_settings set (timescaledb.compress,
+    timescaledb.compress_orderby = 'x',
+    timescaledb.compress_bloom = 'u, u',
+    timescaledb.compress_minmax = 'ts');
```

Comment:
This will be awkward to use with additional index parameters such as false positive rate. Is it complicated to do something like `index  = bloom(device_id, 0.0001), minmax(timestamp)` instead?